Modules/posixmodule.c

```diff
@@ -10058,50 +10058,83 @@ os_times_impl(PyObject *module)
 static PyObject *
 build_itimerspec(const struct itimerspec* curr_value)
 {
+    double _next_expiration = (double)(curr_value->it_value.tv_sec) +
+                    (double)curr_value->it_value.tv_nsec * 1e-9;
```

Comment:
Could this magic number be extracted out?

Reply:
@AA-Turner 
Fixed.